pages/common/sass.md

```diff
@@ -0,0 +1,12 @@
+# Sass
+
+> Converts SCSS or Sass files to CSS
+> .scss and .sass can be used interchangeably in below examples depending on your preference
+
+- Immediately one time convert SCSS or Sass file to CSS
+
+`sass {{inputfile}}.scss:{{outputfile}}.css`
```

Comment:
This is true however the sass command accepts either syntax. For sass --watch, colon is used to distinctly name the output file. Kept with colon syntax to keep consistency between commands.